WorkflowFunctionBundle/Document/Authorization.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace OpenOrchestra\WorkflowFunctionBundle\Document;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
+use OpenOrchestra\WorkflowFunction\Model\AuthorizationInterface;
+
+/**
+ * Description of Authorization
+ *
+ * @ODM\EmbeddedDocument
+ */
+class Authorization implements AuthorizationInterface
+{
+    /**
+     * @var string $name
+     *
+     * @ODM\Field(type="string")
+     */
+    protected $name;
```

Comment:
I disagree with it.

If you set a relation, the authorization will be only linked to the ContentType class, so you would not be able to make it evolve.

If you want to do something more clear, you will need to do some inheritence : 
 - the authorization class as the root
 - a ContentTypeAuthorization class for the authorization on the content types
 - a NodeAuthorization for the authorization on the node

For me the second solution can be better but as a main disavantage, I don't think that the inheritance can be managed by the configuration, can you check this point?